### PR TITLE
feat: Implement ComboBox.IsTextSearchEnabled

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs [hunk 1]
```diff
@@ -13,6 +13,14 @@ public partial class TestServices
 	{
 		public static class KeyboardHelper
 		{
+			static KeyboardHelper()
+			{
+				for (VirtualKey key = VirtualKey.A; key <= VirtualKey.Z; key++)
+				{
+					m_vKeyMapping.Add(char.ToLower((char)key).ToString(), key);
```
Comment (jeromelaban) [inline src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs:20]: Should be build based on the current culture? I think this should be cached and rebuilt on demand.

Comment (Youssef1313) [reply src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs:20]: What would be the benefit of rebuilding on demand? This is executed once and is test-only code anyway.

Comment (jeromelaban) [reply src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs:20]: Ah! My apologies, I missed that it was in tests. It's not needed then.

File: src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs [hunk 2]
```diff
@@ -295,6 +318,32 @@ private void UpdateDescriptionVisibility(bool initialization)
 			}
 		}
 
+		public static DependencyProperty IsEditableProperty { get; } =
```
Comment (MartinZikmund) [inline src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs:321]: As `IsEditable` is not implemented yet, please move these back to generated file

**Review by MartinZikmund — APPROVED** [review]
Awesome!

File: src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs (hunk 2 above)
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs]: Done.

**Comment by Youssef1313** [comment]
@jeromelaban Sure. For this one specifically, it can be seen in any ComboBox sample as IsTextSearchEnabled defaults to true.

**Pull request merged**